src/pages/billing/Project.js

```diff
@@ -17,18 +17,35 @@ import { withErrorReporting } from 'src/libs/error'
 import Events from 'src/libs/events'
 import { FormLabel } from 'src/libs/forms'
 import * as Nav from 'src/libs/nav'
+import { contactUsActive } from 'src/libs/state'
 import * as StateHistory from 'src/libs/state-history'
+import { topBarHeight } from 'src/libs/style'
 import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
 import { billingRoles } from 'src/pages/billing/List'
 
 
 const workspaceLastModifiedWidth = 150
 const workspaceExpandIconSize = 20
+const billingAccountIconSize = 16
 
-const WorkspaceCardHeaders = Utils.memoWithName('WorkspaceCardHeaders', ({ sort, onSort }) => {
+const BillingAccountIcon = {
```

Comment:
Generally, things should only be named with an initial capital if they're React components, i.e. they're intended to be called with `h`. Perhaps this should be `billingAccountIcons`.